diploshic/diploSHIC

```diff
@@ -360,12 +363,25 @@ if argsDict["mode"] == "train":
             np.repeat(4, len(lh1)),
         )
     )
-
     # reshape both to explicitly set depth image. need for theanno not sure with tensorflow
     both = both.reshape(both.shape[0], nDims, numSubWins, 1)
-    if trainingDir == testingDir:
+    if argsDict["domain_adaptation"]:
+        empirical = np.loadtxt(trainingDir + "empirical.fvec", skiprows=1)
+        emp = np.reshape(empirical, (empirical.shape[0], nDims, numSubWins))
+        emp1 = np.concatenate((emp,emp,emp,emp,emp))
```

Comment:
This is the copy 5x line that should be removed in the future when user passes in empirical target domain data the same length of their training set simulations